src/library/scala/concurrent/Future.scala

```diff
@@ -243,7 +243,12 @@ trait Future[+T] extends Awaitable[T] {
    *  this future. If this future is completed with an exception then the new
    *  future will also contain this exception.
    *
-   *  $forComprehensionExamples
+   *  Example:
+   *
+   *  {{{
+   *  val f = Future { "The future" }
+   *  val g = f map { (x: String) => x + " is now!" }
+   *  }}}
```

Comment:
There is nothing wrong with that, but I wanted the example to be as simple to understand as possible.   By explicitly showing `map`, one can very easily see how `map` is being used in the example.